docker-compose-dev.yml

```diff
@@ -0,0 +1,92 @@
+version: '3.7'
+
+services:
+
+  mysql:
+    image: mysql:5.7
+    restart: always
+    ports:
+      - "3306:3306"
+    expose:
+      - "3306"
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: 'yes'
+      MYSQL_DATABASE: 'hotsapi'
+      MYSQL_USER: 'docker'
+      MYSQL_PASSWORD: 'docker'
+    healthcheck:
+      test: ['CMD', '/opt/goss', '-g', '/opt/goss/goss-mysql.yaml', 'validate']
```

Comment:
Should this be `/opt/goss/goss` ? Otherwise it just seems to be pointing at a directory.